docs/run/admission/automation.md

```diff
@@ -7,6 +7,8 @@ meta:
 Fairwinds provides a powerful, flexible solution for fine-grained customization of Admission Controller actions with the [Automation Rules](https://insights.docs.fairwinds.com/features/rules/) feature. 
 
 ## Passive Mode
+** Starting from Insights 6.4.0, the Admission Controller is set to Passive Mode by default for any new clusters **
```

Comment:
I'd put this in the release notes instead, then we can remove this entire section.

Reply:
Updated. Is that what you were thinking?